falco/.gitignore

```
@@ -0,0 +1,2 @@
+charts
+requirements.lock
```

```suggestion
requirements.lock
Chart.lock
```

`Chart.lock` is not strictly needed, we can keep it out. Please, remove the file too.